ms2/src/org/labkey/ms2/MS2Module.java

```diff
@@ -264,41 +262,43 @@ protected void startupAfterSpringConfig(ModuleContext moduleContext)
+            containerFrag.append("SELECT r.Container FROM ");
+            containerFrag.append(MS2Manager.getTableInfoRuns(), "r");
+            containerFrag.append(" WHERE r.Run = ").append(TableUpdaterFileListener.TABLE_ALIAS).append(".Run");
 
-        FileContentService.get().addFileListener(new TableUpdaterFileListener(MS2Manager.getTableInfoFractions(), "MzXMLURL", TableUpdaterFileListener.Type.uri, "Fraction", containerFrag)
-        {
-            @Override
-            public int fileMoved(@NotNull File srcFile, @NotNull File destFile, @Nullable User user, @Nullable Container container)
+            fcs.addFileListener(new TableUpdaterFileListener(MS2Manager.getTableInfoFractions(), "MzXMLURL", TableUpdaterFileListener.Type.uri, "Fraction", containerFrag)
             {
-                int result = super.fileMoved(srcFile, destFile, user, container);
-                MS2Manager.clearFractionCache();
-                return result;
-            }
-        });
-        FileContentService.get().addFileListener(new TableUpdaterFileListener(MS2Manager.getTableInfoProteinProphetFiles(), "FilePath", TableUpdaterFileListener.Type.filePath, "RowId", containerFrag));
-        FileContentService.get().addFileListener(new TableUpdaterFileListener(ProteinSchema.getTableInfoAnnotInsertions(), "FileName", TableUpdaterFileListener.Type.filePath, "InsertId"));
-        FileContentService.get().addFileListener(new TableUpdaterFileListener(ProteinSchema.getTableInfoFastaFiles(), "FileName", TableUpdaterFileListener.Type.filePath, "FastaId"));
+                @Override
+                public int fileMoved(@NotNull File srcFile, @NotNull File destFile, @Nullable User user, @Nullable Container container)
+                {
+                    int result = super.fileMoved(srcFile, destFile, user, container);
+                    MS2Manager.clearFractionCache();
+                    return result;
+                }
+            });
+            fcs.addFileListener(new TableUpdaterFileListener(MS2Manager.getTableInfoProteinProphetFiles(), "FilePath", TableUpdaterFileListener.Type.filePath, "RowId", containerFrag));
+            fcs.addFileListener(new TableUpdaterFileListener(ProteinSchema.getTableInfoAnnotInsertions(), "FileName", TableUpdaterFileListener.Type.filePath, "InsertId"));
+            fcs.addFileListener(new TableUpdaterFileListener(ProteinSchema.getTableInfoFastaFiles(), "FileName", TableUpdaterFileListener.Type.filePath, "FastaId"));
+        }
 
-        UsageMetricsService.get().registerUsageMetrics(getName(), () -> {
-                Map<String, Object> results = new HashMap<>();
-                results.put("hasGeneOntologyData", new TableSelector(ProteinSchema.getTableInfoGoTerm()).exists());
-                return results;
-        });
+        UsageMetricsService.get().registerUsageMetrics(getName(), () -> Map.of("hasGeneOntologyData", new TableSelector(ProteinSchema.getTableInfoGoTerm()).exists()));

```

Comment:
This can probably switch over to the new ProteinModule

Reply:
This has been moved